src/frontend/apps/standalone_site/src/features/Header/Header.tsx

```diff
@@ -76,10 +128,58 @@ const Header = forwardRef<Nullable<HTMLDivElement>>((_props, ref) => {
         <Burger width={60} height={60} aria-controls="menu" />
         <LogoIcon width={117} height={80} />
       </Box>
-      <Box direction="row" align="center" gap="small" justify="end">
-        <Text>{fullName}</Text>
-        <AvatarIcon width={42} height={42} />
-      </Box>
+
+      <DropButton
+        plain
+        label={
+          <Box direction="row" align="center" gap="small" justify="end">
+            <Text>{fullName}</Text>
+            <AvatarIcon width={42} height={42} />
+          </Box>
+        }
+        dropAlign={{ top: 'bottom', right: 'right' }}
+        dropContent={
+          <Box direction="column" margin="small" gap="small">
+            <NavLinkStyled to={routes.PROFILE.path}>
```

Comment:
After the click, is it normal that the dropbox stay open ?

Reply:
Yes and no, I don't like it but it's how the DropButton is designed by grommet and I did not find a way to control this unless not using a DropButton (and that would be quite a shame imo)

Reply:
DropButton has a property `open` to do the switch.

Reply:
According to the documentation it only initiaze the state but does not update it.
`Setting this property does not influence user interaction after it has been rendered.`

Reply:
The documentation is a lie